### PR TITLE
[POL-98] Part 2 of perf test suite setup: running on perf env

### PR DESCRIPTION
**Pull request by snwessel**
This PR is part 2 of [POL-98](https://broadworkbench.atlassian.net/browse/POL-98) (part 1 was [here](https://github.com/DataBiosphere/terra-external-credentials-manager/pull/91)).

These changes enable our performance tests to: 
- be run against the perf deployment of ECM
- authenticate through Sam (and the proxy) using a test user 

It also improves our documentation around how testrunner is used in ECM.

I will make a follow-up PR to automate nightly tests once we determine whether it's possible to access perf from a github workflow ([devops ticket here](https://broadworkbench.atlassian.net/browse/DDO-1879)).

### CODE REVIEW

File: integration/src/main/resources/testusers/scarlett.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "name": "scarlett",
+    "userEmail": "Scarlett.Flowerpicker@test.firecloud.org",
```
Comment (snwessel) [inline integration/src/main/resources/testusers/scarlett.json:3]: This account is used for automated testing in the Terra UI, and it already exists in all of the pre-prod environments (which makes setup a lot easier for us). @ichengchang let me know if there is a different account you would prefer we use for any reason.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
 
-###### Dependency locking is turned on, to update the lockfiles run `./gradlew dependencies --write-locks`
+* [Design Doc](https://docs.google.com/document/d/1vNYCAcyxB5bDwR0KIfKEb4h6kIz52BUGyJTIquzq5bg/edit)
+* [Developer Setup Guide](/DEVELOPMENT.md)
+* [Integration & Performance Testing Guide](/integration/README.md)
+* [Swagger Page (Dev)](https://externalcreds.dsde-dev.broadinstitute.org/)
+
+## Other Notes
+
+Dependency locking is turned on, to update the lockfiles run `./gradlew dependencies --write-locks`
```
Comment (ichengchang) [inline README.md:13]: Nit: Could you also add this note in `integration/README.md`?

File: integration/src/main/resources/testusers/scarlett.json (hunk 1 above)
Comment (ichengchang) [reply integration/src/main/resources/testusers/scarlett.json:3]: This is totally fine. The choice of test users is not mandated by `TestRunner`, developers are free to choose test users that are the most convenient to set up.

File: README.md (hunk 2 above)
Comment (snwessel) [reply README.md:13]: Yeah definitely, thanks for the suggestion!

**Review by ichengchang — APPROVED** [review]
Looks good!

**Review by Shakespeared — APPROVED** [review]
LGTM

**Review by dvoet — APPROVED** [review]

**Pull request merged**